src/functionaltests/resources/features/status.feature

```diff
@@ -0,0 +1,19 @@
+
+@Status
+Feature: Test Status
+
+  @TestStatusIsOk
+  Scenario: Test Status Is AVAILABLE
+    Given "None" services is unavailable
+    When I fetches status
+    Then Verify "eiffelIntelligenceStatus" status is "AVAILABLE"
+    And Verify "rabbitMQStatus" status is "AVAILABLE"
+    And Verify "mongoDBStatus" status is "AVAILABLE"
+    
+  @TestRMQStatusIsUnavailable
+  Scenario: Test RMQ Status Is UNAVAILABLE
```

Comment:
Write out the full name of RabbitMQ here.

Reply:
Done.